src/hooks/use-authentication/index.ts

```diff
@@ -58,28 +85,16 @@ const useAuthentication = (
 		}
 	}, [isAuthenticated, preferencesLoaded])
 
-	// Separating user and session data
-	let session: SessionData, user: UserData
-	if (isAuthenticated) {
-		session = { ...data }
-		user = data.user
-		delete session.user
-	}
-
 	// Return everything packaged up in an object
 	return {
-		error,
-		hasError,
+		error: data?.error,
 		isAuthenticated,
 		isLoading,
-		session,
-		showAuthenticatedUI,
-		showUnauthenticatedUI,
```

Comment:
I want to simplify what is returned from this hook, some of these properties weren't used or can be inferred in other ways. 

The `showAuthenticatedUI` properties were only used in one spot, so to simplify things I decided to remove them from the hook for now, since they can be inferred from other values already returned from the hook.